Library/Formula/libfabric.rb

```diff
@@ -0,0 +1,18 @@
+class Libfabric < Formula
+  desc "OpenFabrics libfabric"
+  homepage "https://ofiwg.github.io/libfabric/"
+  url "http://downloads.openfabrics.org/downloads/ofi/libfabric-1.1.1.tar.bz2"
+  sha256 "228c50d1f9595009a026bb2293f372a04d89ecb3b0f7fcde3905476dbf49fc95"
+  head "https://github.com/ofiwg/libfabric.git"
```

Comment:
You'll need to either remove the `head` or fix it. It won't currently build because they use the autotools to generate the `configure` script so the call to `./configure` further below will just die on `HEAD`.

Just needs something like:

```ruby
head do
  url "https://github.com/ofiwg/libfabric.git"

  depends_on "autoconf" => :build
  depends_on "automake" => :build
  depends_on "libtool"  => :build
  end
```

And then:

```ruby
def install
  system "./autogen.sh" if build.head?
  system "./configure", "--disable-dependency-tracking",
```

And so on.